.scalafmt.conf

```diff
@@ -1,13 +1,52 @@
-version = 2.0.0-RC5
+version = 2.1.0
 
 style = defaultWithAlign
 
-align.tokens               = [off]
-danglingParentheses        = true
 docstrings                 = JavaDoc
 indentOperator             = spray
 maxColumn                  = 120
-rewrite.rules              = [RedundantParens, SortImports]
-spaces.inImportCurlyBraces = true
+rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
 unindentTopLevelOperators  = true
-//binPack.callSite           = true
+align.tokens               = [{code = "=>", owner = "Case"}]
+align.openParenDefnSite    = false
+align.openParenCallSite    = false
+optIn.breakChainOnFirstMethodDot = false
+optIn.configStyleArguments = false
```

Comment:
TBQH I'm not sure if I like this one - this is what forces
```
      val checks = im.Seq(
        SlowCheck,
        OkCheck
      )
```
to be rewritten as
```
      val checks = im.Seq(SlowCheck, OkCheck)
```
While setting this to 'true' seems to allow either (does not touch it)

Reply:
I agree, it seems to depend on where the `)` is placed

Reply:
you could
```
git reset HEAD^
git add .scalafmt.conf project/plugins.sbt
git restore .
sbt scalafmtAll
git commit -a
```
for a smaller diff now ;)

Reply:
I think I did something like that to reformat all with the new version/settings. Do you see anything unexpected?

Reply:
I would not expect https://github.com/akka/akka-management/pull/632/files#diff-b7dade6df5a2268bddde162140416a7fR161 anymore

Reply:
ah, I tried to change those manually. If placing the trailing `)` on a new line (as it was) then it keeps them as it. I was thinking that it looked better with one-liners for these, and misunderstood that you had the opposite opinion. I'll revert those manual changes.

Reply:
Ahh gotcha. I don't have a strong opinion in these particular cases, but in general I like scalafmt not to touch those when I do :smile: